File: Tugas5.java
import java.util.Scanner;

public class Tugas5 {
    public static void main(String[] args) {
        Scanner sumar = new Scanner(System.in);
        System.out.print("Masukan Nilai IPK Anda  = ");
        double IPK = sumar.nextDouble();
        System.out.print("Berapa Tahun Anda Studi Di Sini = ");
        int studi = sumar.nextInt();
        if (IPK >= 3.51 && IPK <= 4 && studi <= 4) {
            System.out.print("Selamat Anda Lulus Summa Cumlaude Dengan IPK = " + IPK + " Dengan Studi Selama " + studi + " Tahun");
        
        } else if (IPK >= 3.51 && IPK <= 4 ) {
            System.out.print("Selamat Anda Lulus Cumlaude Dengan IPK = " + IPK);
        
        } else if (IPK >= 3.01 && IPK < 3.51) {
            System.out.print("Selamat Anda Lulus Sangat Memuaskan Dengan IPK = " + IPK);
        
        } else if (IPK >= 2.76 && IPK < 3.01) {
            System.out.print("Selamat Anda Lulus Memuaskan Dengan IPK = " + IPK);
        
        } else if (IPK >= 2.00 && IPK < 2.76) {
            System.out.print("Selamat Anda lulus Cukup Memuaskan Dengan IPK = " + IPK);
        
        } else if (IPK < 2.00) {
            System.out.print("Mohon Maaf Anda Tidak Lulus Dengan IPK = " + IPK);
            System.out.print(" Tetap Semangat Dan Jangan Menyerah !!!");
        
        } else if  ( IPK > 4 || IPK < 0) {
            System.out.print("Nilai IPK Anda Tidak Valid");
        }
    }
}

File: tugas5.py
ipk = str(input("Masukkan IPK Anda = "))
studi = str(input("Berapa Taun Anda Studi Di Sini = "))

if(float(ipk) >= 3.51 and float(ipk) <= 4 and float(studi) <= 4):
    print("Selamat Anda lulus Summa Cumlaude dengan ipk " + ipk + " dengan studi selama " + studi + " Tahun")

elif(float(ipk) >= 3.51 and float(ipk) <= 4 and float(studi) > 4):
    print("Selamat Anda Lulus Cumlaude Dengan IPK = " + ipk)

elif(float(ipk) >= 3.01 and float(ipk) < 3.51):
    print("Selamat Anda Lulus Sangat Memuaskan Dengan IPK = " + ipk)

elif(float(ipk) >= 2.76 and float(ipk) < 3.01):
    print("Selamat Anda Lulus Memuaskan Dengan IPK = " + ipk)

elif(float(ipk) >= 2.00 and float(ipk) < 2.76):
    print("Selamat Anda lulus Cukup Memuaskan Dengan IPK = " + ipk)

elif(float(ipk) < 2.00):
    print("Mohon Maaf Anda Tidak Lulus Dengan IPK = " + ipk + " Tetap Semangat Dan Jangan Menyerah !!!")

elif(float(ipk) > 4 or float(ipk) < 0):
    print("Nilai IPK Anda Tidak Valid")